src/content/fr/2019/css.md

```diff
@@ -0,0 +1,735 @@
+---
+part_number: I
+chapter_number: 2
+title: CSS
+description: Chapitre CSS du Web Almanac 2019, couvrant les couleurs, unités, sélecteurs, mises en page, typographies et polices, espacements, décoration, animation et media queries.
+authors: [una, argyleink]
+reviewers: [meyerweb, huijing]
+analysts: [rviscomi]
+translators: []
+discuss: 1757
+results: https://docs.google.com/spreadsheets/d/1uFlkuSRetjBNEhGKWpkrXo4eEIsgYelxY-qR9Pd7QpM/
+queries: 02_CSS
```

Comment:
We've moved some more data into the meta data so need these translated and added here:

```suggestion
queries: 02_CSS
una_bio: Una Kravets is a Brooklyn-based international public speaker, technical writer, and Developer Advocate for Material Design at Google. Una hosts the <a href="https://www.youtube.com/watch?v=YK8GZBx3hpg">Designing the Browser</a> web series and the <a href="https://spec.fm/podcasts/toolsday">Toolsday</a> developer podcast. Follow her on <a href="https://twitter.com/una">Twitter</a> to find her musings on creative CSS, user experiences, and web development best practices.
argyleink_bio: Adam Argyle is a Google Chrome developer relations member focused on CSS; He's a web addict with an insatiable lust for great UX & UI; Find him on the web <a href="https://twitter.com/argyleink">@argyleink</a> or checkout his website <a href="https://nerdy.dev">https://nerdy.dev</a>.
featured_quote: Cascading Style Sheets (CSS) are used to paint, format, and layout web pages. Their capabilities span concepts as simple as text color to 3D perspective. It also has hooks to empower developers to handle varying screen sizes, viewing contexts, and printing. CSS helps developers wrangle content and ensure it's adapting properly to the user.
featured_stat_1: 5%
featured_stat_label_1: Pages using custom properties
featured_stat_2: 2%
featured_stat_label_2: Sites that use CSS Grid
featured_stat_3: 780
featured_stat_label_3: Number of digits in largest Z-Index value
```